content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -15,47 +15,63 @@ menu:
 
 # Upgrade from Gogs
 
-Gogs versions up to 0.9.97 can be smootlhy upgraded to Gitea.
+Gogs versions up to 0.9.115 can be smootlhy upgraded to Gitea.
 
-There are some steps to do so below.
-Make sure to complete them as the gitea user in ~:
+There are some steps to do so below. On Unix run as your Gogs user:
 
-* Stop Gogs
-* cd ~
-* wget https://github.com/go-gitea/gitea/archive/v1.0.0.zip
-* unzip v1.0.0.zip
-* cp -r gogs gitea
-* If you have custom content migrated from gogs/custom/ folder, like templates or localization files, you should manually merge these because these files changed on Gitea.
-* cp -r gitea-v1.0.0/options gitea/options
-* cp -r gogs-repositories/ gitea-repositories
-* cp -r gogs-data/ gitea-data/
-* vim gitea/custom/conf/app.ini
+* Create a Gogs backup with `gogs dump`. This creates `gogs-dump-[timestamp].zip` file containing all your Gogs data. 
+* Download the file matching your platform from the [downloads page](https://dl.gitea.io/gitea)
+* Put the binary at the desired install location
+* Copy `gogs/custom/conf/app.ini` to `gitea/custom/conf/app.ini`
+* If you have custom content in `gogs/custom/` copy that to `gitea/custom/options`. This is used to override gitignores, labels, licenses, locales and readmes.
```

Comment:
`gogs/custom/conf` got things like gitignores, licenses, labels and so on which needs to be copied to `gitea/custom/options`. Beside that `gogs/custom/` can also include `templates` and `public` which needs to be copied to `gitea/custom/`.